.github/workflows/test.yaml

```diff
@@ -77,4 +77,4 @@ jobs:
       - name: Generate coverage report
         run: |
           cargo llvm-cov --lcov --all --all-features --all-targets > lcov.info
-          bash <(curl -s https://codecov.io/bash) -N ${{github.sha}} -f lcov.info
+          bash <(curl -s https://codecov.io/bash) -C ${{github.sha}} -f lcov.info
```

Comment:
I don't understand this change. I thought `-C` should point to the top of the branch from the pull-request, but that `github.sha` points to the head of the base branch?